applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -155,6 +155,9 @@ void KratosGeoMechanicsApplication::Register()
     KRATOS_REGISTER_ELEMENT("UPwSmallStrainLinkInterfaceElement3D6N", mUPwSmallStrainLinkInterfaceElement3D6N)
     KRATOS_REGISTER_ELEMENT("UPwSmallStrainLinkInterfaceElement3D8N", mUPwSmallStrainLinkInterfaceElement3D8N)
 
+    KRATOS_REGISTER_ELEMENT("UPw2DLineInterfaceElement2Plus2N", mUPw2DLineInterfaceElement2Plus2N)
+    KRATOS_REGISTER_ELEMENT("UPw2DLineInterfaceElement3Plus3N", mUPw2DLineInterfaceElement3Plus3N)
```

Comment:
We are rethinking the naming convention for element types. The adopted names are a proposal. I would suggest to discuss this with the other team members before merging this branch into `master`. To answer your questions:
- The element's dimension has been moved to the front to avoid having it too close to the number(s) of nodes.
- By the using the "Plus" notation, the name expresses that the interface elements has two sides.
- I don't hope that we have code that relies on the exact naming scheme. If there is, could please point me to it? Thanks.

In addition to the above proposed names, we may consider to prefix these names with `Geo_` to reduce the chances of name clashes with elements from other applications.

Reply:
As far as I remember, there are parts of the code relies on the last part of the naming, namely 2D3N, or 3D4N, etc.
I will search that part(s) today.
I do agree to prefix the names with "Geo" to be consistent with other namings (which are newly added) and avoid possible name clashes.

Reply:
As discussed today, we need to investigate a few more things before we can make an informed decision about the naming scheme to be applied. However, in order to continue with the development of integration tests, we have chosen names that comply with the preliminary scheme.